src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -62,14 +62,15 @@ public class OtlpExporterOptions
 #endif
 
         /// <summary>
-        /// Gets or sets optional headers for the connection.
+        /// Gets or sets optional headers for the connection. Refer to the <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specifying-headers-via-environment-variables">
+        /// specification</a> for information on the expected format for Headers.
         /// </summary>
         public string Headers { get; set; }
 
         /// <summary>
         /// Gets or sets the max waiting time (in milliseconds) for the backend to process each span.
         /// </summary>
-        public int? TimeOut { get; set; }
+        public int? TimeoutMilliseconds { get; set; }
```

Comment:
use 10 secs default, as per spec.

Reply:
Set the default value to 10000 ms and updated the data type to int.